docs/source/data_acquisition/DataObfuscation.rst

```diff
@@ -0,0 +1,578 @@
+Data Obfuscation Functions
+==========================
+
+Sharing data, creating documents and doing public demonstrations often
+require that data containing PII or other sensitive material be
+obfuscated.
+
+MSTICPy contains a simple library to obfuscation data using hashing and
```

Comment:
'to obfuscate'

Reply:
Damn! I hit merge then saw the comment. :-(